src/snowflake/cli/plugins/snowpark/package_utils.py

```diff
@@ -101,86 +103,10 @@ def _get_anaconda_channel_contents():
-            if os.path.exists(record_file_path):
-                # the RECORD file contains a list of files included in the
-                # package, get the unique root folder names and delete them
-                # recursively
-                with open(record_file_path, encoding="utf-8") as record_file:
-                    # we want the part up until the first '/'.
-                    # Sometimes it's a file with a trailing ",sha256=abcd....",
-                    # so we trim that off too
-                    record_entries = list(
-                        {
-                            line.split(",")[0].rsplit("/", 1)[0]
-                            for line in record_file.readlines()
-                        },
-                    )
-                    included_record_entries = []
-                    for record_entry in record_entries:
-                        record_entry_full_path = os.path.abspath(
-                            os.path.join(".packages", record_entry),
-                        )
-                        # it's possible for the RECORD file to contain relative
-                        # paths to items outside of the packages folder.
-                        # We'll ignore those by asserting that the full
-                        # packages path exists in the full path of each item.
-                        if (
-                            os.path.exists(record_entry_full_path)
-                            and packages_full_path in record_entry_full_path
-                        ):
-                            included_record_entries.append(record_entry)
-                    return_dict[package.name] = RequirementWithFiles(
-                        requirement=package, files=included_record_entries
-                    )
-    return return_dict
-
-
-def get_package_name_from_metadata(metadata_file_path: str) -> Requirement | None:
-    """Loads a METADATA file from the dist-info directory of an installed
-    Python package, finds the name of the package.
-    This is found on a line containing "Name: my_package".
-
-    Args:
-        metadata_file_path (str): The path to the METADATA file
-
-    Returns:
-        str: the name of the package.
-    """
-    with open(metadata_file_path, encoding="utf-8") as metadata_file:
-        contents = metadata_file.read()
-        results = re.search("^Name: (.*)$", contents, flags=re.MULTILINE)
-        if results is None:
-            return None
-        requirement_line = results.group(1)
-        results = re.search("^Version: (.*)$", contents, flags=re.MULTILINE)
-        if results is not None:
-            version = results.group(1)
-            requirement_line += f"=={version}"
-        return Requirement.parse(requirement_line)
-
-
 def install_packages(
     file_name: str | None,
     perform_anaconda_check: bool = True,
-    package_native_libraries: PypiOption = PypiOption.ASK,
+    package_native_libraries: PypiOption = PypiOption.YES,
```

Comment:
How does the option look like? I spotted some weir flag like:
```
 --yes   -y        Installs packages that are not available on the Snowflake anaconda channel.
```

I don't think this looks nice. The flag name should be more meaningful than `--yes`

Reply:
There were two different options. `--yes` was resposible for deciding, if we should try to install package from PyPi. I added `--install-from-pip` alias for it, but left `--yes` and `-y` not to break any pipelines using it.
Also added `--allow-native-libraries` to manage this choice

Reply:
Let's make the `yes` a hidden flag with goal to deprecate it. It looks stange.